scss/typography/_dropcaps.scss

```diff
@@ -0,0 +1,64 @@
+////
+/// @group typography-dropcaps
+////
+
+/// Number of lines against the rest of the content.
+/// @type Number
+$dropcaps-lines: 3 !default;
+
+/// Line height for the rest of the content.
+/// @type Number
+$dropcaps-line-height: 1.6 !default;
+
+/// If `true`, makes dropcaps uppercase.
+/// @type Boolean
+$dropcaps-uppercase: true !default;
+
+/// Dropcaps component
+/// @param { Number } $lines [$dropcaps-lines] Number of lines against the rest of the content.
+/// @param { Number } $line-height [$dropcaps-line-height] Line height for the rest of the content.
+/// @param { Boolean } $uppercase [$dropcaps-uppercase] If `true`, makes dropcaps uppercase.
+@mixin dropcaps(
+  $lines: $dropcaps-lines,
+  $line-height: $dropcaps-line-height,
+  $uppercase: $dropcaps-uppercase
+) {
+  // Default margin right (Arial/Helvetica) for having a tiny space between them through the right,
+  // See: http://code.stephenmorley.org/html-and-css/creating-drop-caps
+  $-margin-right: 0.0962em;
```

Comment:
I Have tested with many web fonts like Roboto, Open sans, Nato Sans, Ubuntu and oxygen and works well ! Infact i also tested with monospace and it works there well too. So there is no such font issue as per my best of knowledge and for sure this is not breaking #9596 development

**Although i would also arque that these below private variables should be there as configuration variables so that user's can change it to there font needs !**  

```scss
- $-margin-right 
- $mt
- $mb
```
Thoughts ?